src/expr/impl/src/table_function/generate_series.rs

```diff
@@ -75,17 +75,24 @@ fn generate_series_timestamptz_at_zone(
     generate_series_timestamptz_impl(time_zone, start, stop, step)
 }
 
-#[capture_context(TIME_ZONE)]
 fn generate_series_timestamptz_impl(
     time_zone: &str,
     start: Timestamptz,
     stop: Timestamptz,
     step: Interval,
-) -> Result<impl Iterator<Item = Timestamptz>> {
+) -> Result<impl Iterator<Item = Timestamptz> + use<>> {
```

Comment:
Interesting. So by default it caputures `'_` from `time_zone`, and we are explicitly opting out here?

Reply:
Yes.

Reply:
Wait, I think the change should work for captured version also. Let me bring back the macro